exporter/kafkaexporter/config.go

```diff
@@ -77,6 +77,16 @@ type Producer struct {
 	//   1 -> WaitForLocal. waits for only the local commit to succeed before responding ( default )
 	//   -1 -> WaitForAll. waits for all in-sync replicas to commit before responding.
 	RequiredAcks sarama.RequiredAcks `mapstructure:"required_acks"`
+
+	// Compression Codec used to produce messages
+	// https://pkg.go.dev/github.com/Shopify/sarama@v1.30.0#CompressionCodec
+	// The options are: 'none', 'gzip', 'snappy', 'lz4', and 'zstd'
+	Compression string `mapstructure:"compression"`
```

Comment:
Can you add to `Validate` checking the value is accepted, and a unit test verifying the invalid config?